tune_sklearn/tune_search.py

```diff
@@ -50,15 +52,15 @@ class TuneSearchCV(TuneBaseSearchCV):
             If a list is given, it is sampled uniformly. If a list of dicts is
             given, first a dict is sampled uniformly, and then a parameter is
             sampled using that dict as above.
-            For Bayesian search:
-            If an object is passed in for ``search_optimization``,
-            ``param_distributions`` is ignored because the user specifies
-            the search space.
-            If the string "bayesian" is passed in, ``param_distributions``
-            is a dictionary where keys are
-            parameter names (strings) and values are tuples.
-            Represents search space over parameters of
-            the provided estimator.
+            For Bayesian search: dictionary with parameter names (string)
+            as keys. Values can be
+
+            - a (lower_bound, upper_bound) tuple (for Real
+              or Integer dimensions),
+            - a (lower_bound, upper_bound, "prior") tuple
+              (for Real dimensions),
+            - as a list of categories (for Categorical dimensions), or
```

Comment:
you'll want to link to scikit-optimize documentation somehow

Reply:
https://scikit-optimize.github.io/stable/modules/generated/skopt.Optimizer.html#skopt.Optimizer